### DISCUSSION TITLE
All certificates gone, need to create new api key

### DISCUSSION THREAD

**Discussion by joerndyherrn** [rcl-ssl/docs#49] [General]
Hey there, I just looked up the SSL Certificates Overview and all of my certificates are gone and I need to create a new api key.
A week ago everything was just fine.
This is very urgent as we are running some production ssl certificates using rclapp.

**Comment by anilripla** [comment]
What is the username you use to log into the RCL SSL Portal. I will look into this.

**Comment by joerndyherrn** [comment]
This is sensitive data, but the Subscription Id is xxx

**Comment by anilripla** [comment]
Your certificates are all there for the subscriptions you specified. The domains i found ended with k***el.net. An API Key to access the certificate API was not created. After you create an API Key, you will be able to access the certificate in the portal. 

You can delete the subscription id in this discussion forum if you want.

**Reply by joerndyherrn** [reply]
But it worked without an API key recently - is this a new feature?

**Reply by rcladmin** [reply]
Yes, the RCL SSL portal now works with the RCL SSL API. The portal needs an API Key to access the API. Once you create the key, everything should work fine. Besides the portal, you are now able to programmatically create and renew certificates. Read more here:
https://docs.rclapp.com/api/api.html
https://docs.rclapp.com/sdk/sdk.html

**Comment by joerndyherrn** [comment]
Can confirm that - its working again :)
